src/netbase.cpp

```diff
@@ -132,14 +132,8 @@ std::vector<std::string> GetNetworkNames(bool append_unroutable)
     return names;
 }
 
-static bool LookupIntern(const std::string& name, std::vector<CNetAddr>& vIP, unsigned int nMaxSolutions, bool fAllowLookup, DNSLookupFn dns_lookup_function)
+static std::vector<CNetAddr> LookupIntern(const std::string& name, unsigned int nMaxSolutions, bool fAllowLookup, DNSLookupFn dns_lookup_function)
 {
-    vIP.clear();
-
-    if (!ContainsNoNUL(name)) {
-        return false;
-    }
```

Comment:
I don't think you can remove this check. What @theStack [suggested](https://github.com/bitcoin/bitcoin/pull/26261#pullrequestreview-1265909107) is this, which I agree with:

```diff
-    if (!ContainsNoNUL(name)) {
-        return addresses;
-    }
+    if (!ContainsNoNUL(name)) return {};
```

Reply:
Yes, sorry, I accidentally removed it. I'm pushing it back.